packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -73,6 +73,19 @@ export class TelemetryService {
     }
   }
 
+  public sendGeneralEvent(errorMsg: string, type: string, qty?: string): void {
+    if (this.reporters !== undefined && this.isTelemetryEnabled) {
+      this.reporters.forEach(reporter => {
+        reporter.sendTelemetryEvent('ardGeneralEvent', {
+          extensionName: EXTENSION_NAME,
+          errorMessage: errorMsg,
```

Comment:
is this event emitted for all events including successful events? the errorMessage field indicates these are only for errors, but would like to confirm

Reply:
Good catch! It was a typo, it's supposed to be the subject. I just pushed the change. Thanks!

Reply:
ty!